### PR TITLE
Added fallback to node.data to support server side translations via T…

### PR DESCRIPTION
**Pull request by ghego1**
Fix to make TranslateDirective work server side, see [issue #377](https://github.com/ocombe/ng2-translate/issues/377)

### CODE REVIEW

File: src/translate.directive.ts [hunk 1]
```diff
@@ -95,11 +95,12 @@ export class TranslateDirective implements AfterViewChecked, OnDestroy {
...
                 }
                 if(!node.originalContent) {
-                    node.originalContent = node.textContent;
+                    node.originalContent = node.textContent || node.data;
                 }
                 node.currentValue = isDefined(res) ? res : (node.originalContent || key);
                 // we replace in the original content to preserve spaces that we might have trimmed
-                node.textContent = this.key ? node.currentValue : node.originalContent.replace(key, node.currentValue);
+                if(node.textContent) node.textContent = this.key ? node.currentValue : node.originalContent.replace(key, node.currentValue);
+                else node.data = _this.key ? node.currentValue : node.originalContent.replace(key, node.currentValue);
```
Comment (ocombe) [inline src/translate.directive.ts]: `_this` is undefined, that's why travis fails

**Comment by ocombe** [comment]
Thanks ! Can you fix the error ?

File: src/translate.directive.ts (hunk 1 above)
Comment (ghego1) [inline src/translate.directive.ts]: Oh! copy+paste from working file, my bad...

**Comment by ghego1** [comment]
Done :-)

**Comment by ghego1** [comment]
I can confirm that this issue is still present in v. 6.x.x and that the PR I had submitted still fixes it. Do you know if you will be able to merge?

**Pull request closed without merging**